config/v1/types_apiserver.go

```diff
@@ -88,7 +99,15 @@ type Audit struct {
 	//
 	// If unset, the 'Default' profile is used as the default.
 	// +kubebuilder:default=Default
-	Profile AuditProfileType `json:"profile,omitempty"`
+	// customRules specify profiles per group. These profile take precedence over the
+	// top-level profile field if they apply. They are evaluation from top to bottom and
+	// the first one that matches, applies.
+	//
+	// +listType=map
+	// +listMapKey=group
```

Comment:
something is wrong here. Either the syntax of these markers is different or our controller-tools version is old. @EmilienM can you check in kubebuilder docs how to specify list types?

Reply:
Oh wow, I did not know there was another RH with a Git handle this close to mine.

Reply:
> Oh wow, I did not know there was another RH with a Git handle this close to mine. 

hi 😊

Reply:
I believe maps is still valid:
https://book.kubebuilder.io/reference/markers/crd-processing.html?highlight=listType#crd-processing

what are your thoughts?

Reply:
Probably our controller-gen is too old, see https://github.com/openshift/build-machinery-go/blob/master/make/targets/openshift/controller-gen.mk#L11. Sync with @damemi how to update it.

Reply:
@sttts do we still need our fork of controller-tools? Last update was in 2019 and the last release you published in February https://github.com/openshift/kubernetes-sigs-controller-tools/releases/tag/v0.2.5

Reply:
Also it could be a type error.

Reply:
@damemi probably we only need it to host the binaries.